pkg/storageos/delete.go

```diff
@@ -78,6 +80,17 @@ func (s *Deployment) Delete() error {
 		}
 	}
 
+	// Delete cluster role for openshift security context constraints.
+	if strings.Contains(s.stos.Spec.K8sDistro, k8sDistroOpenShift) {
```

Comment:
I like using the k8sDistro here, but how does it get set?

Reply:
We have to set it in the cluster config while deploying.
We should change the CR example for cluster to have k8sDistro set to `openshift` for rhel operator release.